src/main/webapp/js/single-conversation-topic.js

```diff
@@ -0,0 +1,153 @@
+// get query param
+// when fetching messages, pass that query param as a query param
+
+const urlParams = new URLSearchParams(window.location.search);
+const parameterConversationTopicId = urlParams.get("id");
+const parameterConversationTopicTitle = urlParams.get("title");
+
+console.log(parameterConversationTopicId);
+
+function setConversationTopicTitle() {
```

Comment:
Why did you choose to iterate through all known topic titles instead of just using the parameterConversationTopicTitle?